server/db/config/connection.js

```diff
@@ -1,12 +1,17 @@
 const { Pool } = require('pg')
+// will not have access to app.js for every test ie. testing database queries 
+require("dotenv").config();
 
-const connectionString = process.env.DATABASE_URL;
+let connectionString =
+  process.env.NODE_ENV === "test" ?
+  process.env.TEST_DATABASE_URL :
+  process.env.DATABASE_URL;
 
 if (!connectionString) throw new Error('DATABASE_URL is not found!');
 
 const options = {
   connectionString,
-  ssl: true,
+  ssl: !connectionString.includes("localhost")
```

Comment:
you can do this 
```suggestion
  ssl: true
```

Reply:
afraid I can't when I set it to true I kept getting this when trying to run the tests
![](https://i.imgur.com/o2sbP3v.png)

Local databases dont always support SSL